src/components/NavBar/index.tsx

```diff
@@ -44,18 +50,32 @@ const NavigationBar = () => {
         <IconButton aria-label="Edit" icon={<BiPencil />} variant="navbarIconBtn" />
       </Flex>
       <Flex alignItems="center" justifyContent="flex-end">
-        <IconButton
-          aria-label="User information"
-          icon={<FaRegUser />}
-          variant="navbarIconBtn"
-          bg="white"
-          color="black"
-          marginRight="10px"
-          isRound
-        />
+        <Menu>
+          <MenuButton
+            as={IconButton}
+            aria-label="User information"
+            icon={<FaRegUser />}
+            variant="navbarIconBtn"
+            bg="white"
+            color="black"
+            marginRight="10px"
+            isRound
+          ></MenuButton>
+          <MenuList>
+            <MenuItem onClick={onOpen} data-testid="login-btn">
+              log in
+            </MenuItem>
+            <MenuItem onClick={onOpen} data-testid="logout-btn">
+              sign up
+            </MenuItem>
+          </MenuList>
```

Comment:
No hover effect on this list(log in and sign up)

Reply:
add the hover property